app/views/propertyDetails/confirmAddress.scala.html

```diff
@@ -40,15 +40,23 @@
         </ul>
     </div>
 
-    <div class="form-group">
+    <div class="margin-top-default">
         <a href="@controllers.propertyDetails.routes.PropertyDetailsAddressController.view(id, fromConfirmAddressPage = true, periodKey, mode)"
         id="edit-address-link"
         data-journey-click="ated-fronted:click:manual-entry"
-        onkeyup='spaceBarHandler(event,"@controllers.propertyDetails.routes.PropertyDetailsAddressController.view(id, fromConfirmAddressPage = true, periodKey, mode)")'>
-        <p id="editAddress">@Messages("ated.confirm-address.editAddress")</p></a>
+        onkeyup='spaceBarHandler(event, "@controllers.propertyDetails.routes.PropertyDetailsAddressController.view(id, fromConfirmAddressPage = true, periodKey, mode)")'>
+            <p id="editAddress">@Messages("ated.confirm-address.editAddress")</p>
+        </a>
 
-    @form(action=controllers.propertyDetails.routes.ConfirmAddressController.submit(id, periodKey, mode)) {
-        <p><button class="button" id="submit" type="submit">@Messages("ated.confirm-and-continue")</button></p>
-    }
+        <a href="@controllers.routes.ExistingReturnQuestionController.view(periodKey, "charge")">
```

Comment:
Please check with Liz where the link is suppose to go to (as discussed yesterday).